### PR TITLE
handle error in finding cpu info #17

### PR DESCRIPTION
**Pull request by giulianogiari**
print an informative message if cpuinfo.get_cpu_info() has an error and stop execution

### CODE REVIEW

File: src/utils/cpu.py [hunk 1]
```diff
@@ -13,12 +13,12 @@ def get_cpu_info() -> str:
...
     """
-
     try:
         cpu_info = cpuinfo.get_cpu_info()["brand_raw"]
-    except:
-        cpu_info= "CPU Info not available"
-        
+    except Exception as e:
+        print("Error retrieving CPU info. Stopping execution.")
+        sys.exit(1)
```
Comment (FrnRmn) [inline src/utils/cpu.py:20]: Looks fine to me!

**Review by FrnRmn — COMMENTED** [review]
Let's wait for the other merges to rebase it maybe?

**Review by MattiaSilvestri — APPROVED** [review]
Fine for me as well.

**Review by FrnRmn — APPROVED** [review]
I rebased with the new dev

**Pull request merged**